x-pack/plugins/files/public/components/file_picker/components/file_card.scss

```diff
@@ -0,0 +1,5 @@
+.filesFilePicker {
+  .euiCard__content, .euiCard__description {
+    margin :0; // make the cards a little bit more compact
```

Comment:
Not that I don't agree with you. But I'm not sure there's a significant reason to overwrite our EUI styles here.  Ideally we wouldn't need to do this to save future maintenance (and work for you)